### PR TITLE
Fix CookieResponse and CloseWindowResponse serialization.

### PR DESCRIPTION
**Pull request by jgraham**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/webdriver-rust/79)

### CODE REVIEW

File: src/response.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ pub enum WebDriverResponse {
 impl WebDriverResponse {
     pub fn to_json_string(self) -> String {
         let obj = match self {
-            WebDriverResponse::CloseWindow(ref x) => json::encode(x),
+            WebDriverResponse::CloseWindow(ref x) => json::encode(&x.to_json()),
```
Comment (andreastt) [inline src/response.rs]: What did this do before?

File: src/response.rs [hunk 2]
```diff
@@ -34,7 +34,8 @@ impl WebDriverResponse {
         }.unwrap();
 
         match self {
-            WebDriverResponse::Generic(_) => obj,
+            WebDriverResponse::Generic(_) |
+            WebDriverResponse::Cookie(_) => obj,
```
Comment (andreastt) [inline src/response.rs]: Should this not be wrapped in `{value: ["window handle 1", "window handle 2"]}`?

File: src/response.rs [hunk 3]
```diff
@@ -200,3 +220,104 @@ impl CookieResponse {
...
+    fn test_cookie() {
+        let resp = WebDriverResponse::Cookie(CookieResponse::new(
+            vec![
+                Cookie::new("test".into(),
+                            "test_value".into(),
+                            Nullable::Value("/".into()),
+                            Nullable::Null,
+                            Nullable::Null,
+                            true,
+                            false)
```
Comment (andreastt) [inline src/response.rs]: This is a good example why I don’t like the `::new` constructors so much. It’s clearer what each argument is when using a struct literal:

```rust
Cookie {
    name: "test".into(),
    value: "test_value".into(),
    path: Nullable::Value("/".into()),
    …
}
```

Anyway, you don’t have to change this.

File: src/response.rs (hunk 1 above)
Comment (jgraham) [inline src/response.rs]: Gave a message like
```
{ "value": {"window_handles": ["test"]}}
```

File: src/response.rs (hunk 2 above)
Comment (jgraham) [inline src/response.rs]: Did you mean to comment on this line?

**Pull request merged**